ruby/ql/lib/change-notes/2023-02-03-twirp.md

```diff
@@ -0,0 +1,4 @@
+---
+ category: minorAnalysis
+---
+* Support for [Twirp framework](https://twitchtv.github.io/twirp/docs/intro.html).
```

Comment:
We should probably add a few more words to describe which parts of Twirp are modelled here.